src/main/kotlin/org/ionproject/integration/domain/common/Validator.kt

```diff
@@ -0,0 +1,7 @@
+package org.ionproject.integration.domain.common
+
+class Validator {
```

Comment:
is this really necessary?

also, a class not needed, we can define top level functions without a holding class.

Reply:
The validator could be useful for other BOs or DTOs and place all the rules for the model. As we discussed with the supervisor, some institutions can have more terms that just the 2 that we are used to, so this is not needed.

Reply:
what I mean is that we may not need an entire class to hold a single function. 

the class itself is not needed (we could have just a top level function) and maybe we could find a reasonable place to put the function in, unless more similar functions were to be moved / added to this file.